crates/bevy_asset/src/processor/mod.rs

```diff
@@ -1,3 +1,25 @@
+//! Asset processing in Bevy is a framework by which artist-authored assets can be automatically prepared for use in game.
+//!
+//! You can think of the asset processing system as a "build system" for assets.
+//! When an artist adds a new asset to the project, the asset processing system will automatically perform the specified processing steps on the asset.
+//! This can include things like converting a `.psd` file to a `.png`, compressing a `.wav` file to an `.ogg`, or generating mipmaps for a texture.
+//!
+//! Its core values are:
+//!
+//! 1. Automatic: changing the artist-format asset should not require any manual steps to get it back in the game.
```

Comment:
```suggestion
//! 1. Automatic: changing the artist-format asset should not require any manual steps ... to get it back in the game.
```

Everything after `...` seems weird to me, try to find a different phrase. Not entirely sure what you want to say here, do you mean "no code modifications to your project are needed to use it in game"?

Not sure if that's necessarily always true.

Reply:
Basically, the ideal here is "artist follows simple rules to edit the source file" -> "Bevy handles the process very easily" -> "changes are visible in the game". No random bash scripts or manual file format conversion needed.